cmd/pebble/cmd_version_test.go

```diff
@@ -35,4 +35,8 @@ func (s *PebbleSuite) TestVersionCommand(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(s.Stdout(), Equals, "client  4.56\nserver  7.89\n")
 	c.Assert(s.Stderr(), Equals, "")
+
+	rest, err := pebble.Parser(pebble.Client()).ParseArgs([]string{"version", "extra", "args"})
```

Comment:
Once again, let's make this a separate test, `TestVersionExtraArgs` or something.